task/sonarqube-scanner/0.1/sonarqube-scanner.yaml

```diff
@@ -35,6 +36,10 @@ spec:
       script: |
         #!/usr/bin/env bash
 
+        if [ ! -f $(workspaces.sonar-secret.path)/token ]; then
```

Comment:
Could you also please add validation to check if workspace is bounded (as we are using optional workspace)
Something like: 
```
 if [ "$(workspaces.sonar-secret.bound)" == "true" ]
```

Reply:
👍